README.md

```diff
@@ -1,9 +1,47 @@
 # ansible-documentation
-Clone of ansible/ansible with non-docs content removed
 
-Before you build the docs locally, you need to:
+This repository holds the ReStructuredText (RST) source, and other files, for user documentation related to the Ansible package and Ansible core.
 
-```text
-python3 docs/bin/clone-core.py
-```
+> Documentation for modules and plugins that are officially supported by the Ansible core engineering team is available in the [`ansible/ansible`](https://github.com/ansible/ansible) repository.
 
+## Building Ansible community documentation
+
+Complete the following steps to build Ansible community documentation from this repository:
+
+* Optionally create a Python virtual environment.
+
+    ```bash
+    # Create a virtual environment in your current directory.
+    python3 -m venv venv
+
+    # Activate the virtual environment to start using it.
+    source venv/bin/activate
+    ```
+
+1. Set up your environment.
+
+    ```bash
+    #
+    python3 -m pip install --upgrade pip setuptools six wheel
```

Comment:
I don't think this step is required.

Reply:
While not strictly necessary I think it's nice to have. It might also help avoid problems/questions from warnings like this one:

```
DEPRECATION: sh is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed.
```

Reply:
been here 5 yrs. never did this step.

Reply:
I'd drop this step. The only part of it that seems relevant is `wheel`, and that's only needed to eliminate a warning from an indirect dependency (`sh`) -- which if that dependency goes away or has its packaging updated, is no longer helpful. Having fewer steps to get to a working environment seems like a good idea.

Reply:
@oraNod Matt is right: only `wheel` is needed to address that warning. Though, updating `pip` is also a good idea for newer platforms, tags and ABI3 wheels so pip wouldn't fall back to building from sdist in some cases.

Reply:
@webknjaz Thx. As it turns out this is pretty much covered in the docs and I was reinventing the wheel.